Example/Leanplum-SDK/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -84,10 +84,15 @@
       "idiom" : "ipad",
       "size" : "83.5x83.5",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ios-marketing",
```

Comment:
Whats this image for?

Reply:
no idea how that got committed, will remove it.